docs/content/commands/pack.md

```diff
@@ -72,9 +72,12 @@ paket pack nugets --symbols
 ### Including referenced projects
 
 Paket automatically replaces inter-project dependencies with NuGet dependencies
-if the dependency has it's own [`paket.template`][templatefile]. In addition to
-this the parameter `--include-referenced-projects` instructs Paket to add
-project output to the package for inter-project dependencies that don't have a
+if the dependency has it's own [`paket.template`][templatefile]. Version constraint 
+for this dependencies can be controlled by value of `--interproject-references` 
```

Comment:
Please fix spelling:

for *these* dependencies can be controlled *with the* `--interproject-references`

(Change "these" or "dependencies", depending on my previous comment.)